test/auro-input.test.js

```diff
@@ -228,13 +228,24 @@ describe('auro-input', () => {
   describe('handles i18n', () => {
     it('credit-card translation', async () => {
       const el = await fixture(html`
+        <auro-input type="credit-card" required id="input01"></auro-input>
+      `);
+
+      const eli18n = await fixture(html`
         <auro-input type="credit-card" required lang="es" id="input01"></auro-input>
       `);
 
-      expect(el.shadowRoot.querySelector('#input01')).to.have.attribute('lang', 'es');
+      const eli18nContent = eli18n.shadowRoot.querySelectorAll('[name="helptext"]')[0].innerHTML;
+      const content = el.shadowRoot.querySelectorAll('[name="helptext"]')[0].innerHTML;
```

Comment:
nit: just use `querySelector` here if we're getting the first element
```suggestion
      const eli18nContent = eli18n.shadowRoot.querySelector('[name="helptext"]').innerHTML;
      const content = el.shadowRoot.querySelector('[name="helptext"]').innerHTML;
```